docs/blog/2019-08-07-theme-jam/index.md

```diff
@@ -0,0 +1,92 @@
+
+## How were entries reviewed?
+
+Every Theme Jam entry needed to have a demo site, a published npm package, a public repo, and a properly configured `package.json`.
+
+Entries also needed to be incredibly performant (something something blazing fast) and meet accessibility standards.
+
+On top of that, submissions were code reviewed for documentation, the ability to extend and shadow the themes, and creativity.
+
+### Part 1: automated review
+
+Several parts of the judging could be done automatically, so we set up an ugly-but-effective [automated test](https://github.com/jlengstorf/theme-jam-tester) that would:
+
+1. Run a [webpagetest.org](https://webpagetest.org/) audit on the demo site
+2. Run a [Lighthouse](https://developers.google.com/web/tools/lighthouse/) audit on the demo site
+3. Check for required fields and keywords in `package.json` (shout out to [unpkg.com](https://unpkg.com) for making this easy)
+
+To pass the automated review, a theme must:
+
+- score at least 90 on the Lighthouse accessibility audit
+- score at least 80 on the Lighthouse performance audit
+- have all required fields in `package.json`
+
+Of the 111 submissions, 52 passed the automated review.
+
+### Part 2: manual review
+
+Every theme that cleared the automated review was manually reviewed by [Jackson](https://twitter.com/jxnblk), [Amberley](https://twitter.com/amber1ey), [John](https://twitter.com/4lpine), [Kyle Gill](https://gill_kyle), [Chris](https://twitter.com/chrisbiscardi), [Flo](https://twitter.com/fk), and me.
+
+Amberley did a manual accessibility audit. All of the themes we reviewed has a few accessibility issues, so we will be opening issues with feedback and suggestions to improve that.
+
+We also looked for coding best practices, ease of [shadowing](/blog/2019-04-29-component-shadowing/)/extending the theme, documentation in the README, and overall creativity/ambitiousness of the project.
+
+## Highlighting Exemplary Creators
+
+We plan to feature a lot of the excellent creators and their themes that were created both as part of Theme Jam and outside of it.
+
+For Theme Jam, we specifically wanted to call out excellence. This is really important with themes being so early; great examples lead to improving the quality of the entire ecosystem.
+
+Two of the themes we received stood out especially. As we reviewed, we all took notes on the themes we thought were outstanding, and _every one of us_ chose these two themes as excellent examples.
+
+**So please join us in congratulating Vojtěch Holík and Allan Pooley, who will receive an all-expenses paid trip to Gatsby Days in recognition of their outstanding work on [`gatsby-theme-simplecast`](https://github.com/vojtaholik/gatsby-theme-simplecast) and [`gatsby-theme-legals-prismic`](https://github.com/littleplusbig/gatsby-theme-legals-prismic)!**
```

Comment:
```suggestion
**So please join us in congratulating Vojtěch Holík and Allan Pooley on their outstanding work on [`gatsby-theme-simplecast`](https://github.com/vojtaholik/gatsby-theme-simplecast) and [`gatsby-theme-legals-prismic`](https://github.com/littleplusbig/gatsby-theme-legals-prismic)!**
```

Reply:
I (personally) don't really see a need to highlight Gatsby Days again, the focus is on their work.